forward_script/src/QueryBuilder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace Wikimedia\ForwardScript;
+
+class QueryBuilder {
+
+	public function getQuery( $info, $pageName, $id = '', $coordinates = [] ) {
+		$category = preg_replace( '/^(:?Category|Kategorie):/', '', $info->category );
```

Comment:
Just wondering, not suggesting to change it: Any reason to use non-capturing group in the regular expression here?